gobblin-core/src/main/java/gobblin/source/extractor/extract/restapi/RestApiExtractor.java

```diff
@@ -148,8 +155,11 @@ public void extractMetadata(String schema, String entity, WorkUnit workUnit)
           if (inputQuery == null || columnListInQuery == null || (columnListInQuery.size() == 1 && columnListInQuery
               .get(0).equals("*")) || (columnListInQuery.size() >= 1 && this
               .isMetadataColumn(columnName, columnListInQuery))) {
-            this.columnList.add(columnName);
-            columnArray.add(jsonObject);
+             if (!columnListExcluded.contains(columnName.trim().toLowerCase()))
+             {
+        		   this.columnList.add(columnName);
+        		   columnArray.add(jsonObject);
+      		  }
```

Comment:
This line has a different indentation, and it seems it contains tabs. Could you make sure your code conforms to LinkedIn's coding conventions (https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Java+Coding+Conventions) by downloading the XML and use it in Eclipse.